study1/week1/Third Maximum Number.java

```diff
@@ -0,0 +1,30 @@
+class Third Maximum Number {
+
+    int count = 3;            
+    int tmp = Integer.MIN_VALUE;
+
+    public int thirdMax(int[] nums) {              // Space Complexity= O(n), n is the length of the Array
+      Arrays.sort(nums);                          // time complexity = O(n * log(n))
+      if(nums.length <3){    
+          return nums[nums.length-1];
+      }else{
```

Comment:
Regarding indentation this [article](https://testing.googleblog.com/2017/06/code-health-reduce-nesting-reduce.html) explains why it's helpful to keep indentation levels lower.

Keeping here an else statement or not doesn't make any difference in semantics, thus it's better to get rid of this else statement as the if statement just returns from the function right away.